frontend/cypress/integration/admin/admin-smoke-tests.spec.js

```diff
@@ -0,0 +1,68 @@
+import {
+  aliasMutation,
+  aliasQuery,
+  setUpCommonGraphqlIntercepts,
+} from "../../support/graphql-test-utils";
+import { getInputByLabel } from "../../support/helpers";
+
+describe("Admin Smoke Tests", () => {
+  const loginViaUI = (role) => {
+    cy.fixture("users.json").then((users) => {
+      const user = users[role];
+      cy.get("input[name=username]").type(user.email);
+    });
+    cy.findByText("Sign-in").click();
+  };
+
+  beforeEach(() => {
+    // common requests
+    setUpCommonGraphqlIntercepts();
+    // page specific requests
+    cy.intercept("POST", "/graphql", (req) => {
+      aliasQuery(req, "AllUsers");
+      aliasQuery(req, "User");
+      aliasMutation(req, "UpdateUserAsAdmin");
+    });
+
+    cy.visit("/en/admin");
+  });
+
+  it("Searches for a user, reviews the profile, and edits the phone number", () => {
+    // hit the home page, login, hit the dashboard
+    cy.contains("h1", "Home");
+    cy.contains("a", "Login").click();
+    loginViaUI("admin");
```

Comment:
```suggestion
    cy.login("admin");
```

Reply:
Applied in https://github.com/GCTC-NTGC/gc-digital-talent/pull/3084/commits/383b3126d8c6c36e4ef791e4cb651144a270797b